### PR TITLE
Modifying all 6 files related to this parameter

### PR DESCRIPTION
**Pull request by j0rt3g4**
PReviosly I modified just the one the person asked in the issue.
so this will be a different issue. #1071

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/addsadministration/set-adserviceaccount.md [hunk 1]
```diff
@@ -745,23 +745,45 @@ Accept wildcard characters: False
...
 
 ### -ServicePrincipalNames
-Use the **DateTime** syntax when you specify this parameter.
-Time is assumed to be local time unless otherwise specified.
-When a time value is not specified, the time is assumed to 12:00:00 AM local time.
-When a date is not specified, the date is assumed to be the current date.
+Specifies the service principal names for the account.
+This parameter sets the **ServicePrincipalNames** property of the account.
+The LDAP display name (**ldapDisplayName**) for this property is servicePrincipalName.
+This parameter uses the following syntax to add remove, replace or clear service principal name values.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-adserviceaccount.md]:
```suggestion
This parameter uses the following syntax to add, remove, replace or clear service principal name values.
```

File: docset/windows/addsadministration/new-adserviceaccount.md [hunk 2]
```diff
@@ -788,6 +788,12 @@ For example, use the following syntax to add and remove service principal names.
 - Add
 - Replace
 
+The following example shows how to add and remove service principal names.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/new-adserviceaccount.md]:
```suggestion
The following example shows how to add and remove service principal names{
```

File: docset/windows/addsadministration/set-adserviceaccount.md [hunk 3]
```diff
@@ -745,23 +745,45 @@ Accept wildcard characters: False
...
+
+`@{Add=value1,value2,...};@{Remove=value3,value4,...}`
+
+The operators are applied in the following sequence: 
+
+- Remove
+- Add
+- Replace
+
+The following example shows how to add and remove service principal names.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-adserviceaccount.md]:
```suggestion
The following example shows how to add and remove service principal names:
```

File: docset/winserver2012-ps/activedirectory/New-ADServiceAccount.md [hunk 4]
```diff
@@ -872,47 +872,39 @@ This parameter sets the **ServicePrincipalNames** property of the account.
...
-The operators will be applied in the following sequence:
+The operators are applied in the following sequence: 
 
-..Remove
+- Remove
+- Add
+- Replace
 
-..Add
+The following example shows how to add and remove service principal names.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADServiceAccount.md]:
```suggestion
The following example shows how to add and remove service principal names:
```

File: docset/winserver2012r2-ps/addsadministration/New-ADServiceAccount.md [hunk 5]
```diff
@@ -783,6 +783,12 @@ The operators are applied in the following sequence:
 - Add
 - Replace
 
+The following example shows how to add and remove service principal names.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/New-ADServiceAccount.md]:
```suggestion
The following example shows how to add and remove service principal names:
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Final approval needed for this PR that has been discussed earlier. Thanks.

**Comment by dariomws306** [comment]
I think parameter type on Set-ADServiceAccount.md should be HashTable.

Then I am ok.

**Comment by tseward** [comment]
Agreed. If the parameter values are operation={value1,value2...}, that is a hash table.

```powershell
08:53:16| C:\Users\TrevorSeward>$v = @{Add='value1','value2'}
08:56:20| C:\Users\TrevorSeward>$v.GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Hashtable                                System.Object
```

**Comment by j0rt3g4** [comment]
Will. Fix later today

**Comment by j0rt3g4** [comment]
it's done

**Comment by dariomws306** [comment]
Thank you @j0rt3g4, looks good now!

**Comment by e0i** [comment]
@AndreaBarr This PR can now be merged .Thanks.

**Pull request merged**